src/applications/check-in/day-of/pages/Confirmation/CheckInConfirmation.jsx

```diff
@@ -129,7 +126,10 @@ const CheckInConfirmation = props => {
         pageTitle={t('youre-checked-in')}
         testID="multiple-appointments-confirm"
       >
-        <p className="vads-u-font-family--serif">{t('your-appointment')}</p>
+        <p data-testid="confirmation-message">
+          {t('well-come-get-you-from-the-waiting-room')}
```

Comment:
This text is different than what is in the figma file. Am I looking at the wrong thing?

I see this:
![Screenshot 2024-01-17 at 3 38 01 PM](https://github.com/department-of-veterans-affairs/vets-website/assets/13967174/72ded700-2ba7-417c-bff6-90d9a4514cb9)

Reply:
This is odd. I see the correct text and line 130 is the correct key for it.
![Screenshot 2024-01-18 at 11 45 50 AM](https://github.com/department-of-veterans-affairs/vets-website/assets/2982977/ba7990b7-6c3d-4bd6-b377-9e33d8099815)

Reply:
what I screenshot is what is in figma, not what I am seeing in the app